tests/backup/backup_restore_basic_test.go

```diff
@@ -2800,3 +2802,410 @@ var _ = Describe("{BackupRestoreOnDifferentK8sVersions}", func() {
+			log.InfoD("creating source and destination cluster")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+
+			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
+			dash.VerifyFatal(err, nil, "create source and destination cluster")
+
+			clusterStatus, err := Inst().Backup.GetClusterStatus(orgID, SourceClusterName, ctx)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", SourceClusterName))
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", SourceClusterName))
+
+			scheduledClusterUid, err = Inst().Backup.GetClusterUID(ctx, orgID, SourceClusterName)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching [%s] cluster uid", SourceClusterName))
+
+			clusterStatus, err = Inst().Backup.GetClusterStatus(orgID, destinationClusterName, ctx)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", destinationClusterName))
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", destinationClusterName))
+		})
+
+		Step("verify kubernetes version of source and destination cluster", func() {
+			var srcVer, destVer semver.Version
+			log.InfoD("begin verification kubernetes version of source and destination cluster")
+
+			defer func() {
+				log.InfoD("switching to default context")
+				err := Inst().S.SetConfig("")
+				log.FailOnError(err, "failed to SetClusterContext to default cluster")
+			}()
+
+			Step("Get kubernetes source cluster version", func() {
+				log.InfoD("switched context to source")
+
+				sourceClusterConfigPath, err := GetSourceClusterConfigPath()
+				log.FailOnError(err, "failed to get kubeconfig path for source cluster. Error: [%v]", err)
+
+				err = Inst().S.SetConfig(sourceClusterConfigPath)
+				log.FailOnError(err, "failed to switch to context to source cluster [%v]", sourceClusterConfigPath)
+
+				ver, err := k8s.ClusterVersion()
+				log.FailOnError(err, "failed to get source cluster version")
+				srcVer, err = semver.Make(ver)
+				log.FailOnError(err, "failed to get source cluster version")
+			})
+
+			Step("Get kubernetes destination cluster version", func() {
+				log.InfoD("switched context to destination")
+
+				destinationClusterConfigPath, err := GetDestinationClusterConfigPath()
+				log.FailOnError(err, "failed to get kubeconfig path for destination cluster. Error: [%v]", err)
+
+				err = Inst().S.SetConfig(destinationClusterConfigPath)
+				log.FailOnError(err, "failed to switch to context to destination cluster [%v]", destinationClusterConfigPath)
+
+				ver, err := k8s.ClusterVersion()
+				log.FailOnError(err, "failed to get destination cluster version")
+				destVer, err = semver.Make(ver)
+				log.FailOnError(err, "failed to get destination cluster version")
+			})
+
+			log.InfoD("Compare Source and Destination cluster version numbers")
```

Comment:
Why do you need this InfoD?

Reply:
part of old implementation.

✅ removed